magenta/models/shared/melody_rnn_generate.py

```diff
@@ -125,13 +126,21 @@ def run_with_flags(melody_rnn_sequence_generator):
   if not os.path.exists(FLAGS.output_dir):
     os.makedirs(FLAGS.output_dir)
 
+
+#primer_sequence from written melody or given midi file or random note between midi #48 = 4C to midi #84 = 7C
```

Comment:
Is it working for you? I get this error when I don't include the primer in the command line

>     bpm = (primer_sequence.tempos[0].bpm if primer_sequence.tempos
> AttributeError: 'NoneType' object has no attribute 'tempos'
> ERROR: Non-zero return code '1' from command: Process exited with status 1.